packages/manager-components/package.json

```diff
@@ -64,7 +65,11 @@
     "eslint-plugin-storybook": "0.6.13",
     "file-loader": "^6.2.0",
     "husky": "8.0.3",
+    "is-ip": "5.0.1",
+    "is-cidr": "5.0.5",
```

Comment:
it shouldn't be in dev deps if you are importing the lib in your code

also could you use a single lib to handle all ip validation needs ?